test/ts/migrations/4_deploy_terms_contracts_test.ts

```diff
@@ -0,0 +1,64 @@
+// External Libraries
+import * as chai from "chai";
+
+// Utils
+import { Web3Utils } from "../../../utils/web3_utils";
+import ChaiSetup from "../test_utils/chai_setup";
+
+// Wrappers
+import { SimpleInterestTermsContractContract as SimpleInterestTermsContract } from "../../../types/generated/simple_interest_terms_contract";
+
+import { CollateralizedSimpleInterestTermsContractContract as CollateralizedTermsContract } from "../../../types/generated/collateralized_simple_interest_terms_contract";
+
+import { IncompatibleTermsContractContract as IncompatibleTermsContract } from "../../../types/generated/incompatible_terms_contract";
+
+import { CollateralizerContract as Collateralizer } from "../../../types/generated/collateralizer";
```

Comment:
Style nit: please remove spaces between each import that is of the same category.

Reply:
Yeah, the prettier is actually doing this automatically, which is annoying.

Since it's a long line, I'm trying to write it as:
```javascript
import {
  x..........................
} from "y.......................";
```